### ISSUE TITLE
Ideas for improvements to component status page

### ISSUE THREAD

**Issue by danoc** [thumbtack/thumbprint#226]
- [ ] Use blue for "In progress" and a gray circle for "To-do" since the current yellow could be interpreted as a warning and the existing "To-do" blue stands out more than the "In progress" yellow.
- [ ] Link to the component documentation from that page.
- [ ] Link to JIRA tickets and GitHub issues for to-do's and in progress work.
- [ ] Get the data from front-matter in MDX files rather than hard-coding it all in that one MDX file.

**Comment by danoc** [comment]
@tomgenoni — Anything here you think I'm missing? Or something that is here but shouldn't be?

**Comment by tomgenoni** [comment]
> Link to JIRA tickets...

Maybe this could be included in the front-matter? An enhancement could be to show the status of the ticket. We might need/want to support multiple tickets.

**Comment by danoc** [comment]
> > Link to JIRA tickets...
> 
> Maybe this could be included in the front-matter? An enhancement could be to show the status of the ticket. We might need/want to support multiple tickets.

Yeah, we can add to the front-matter. Although, until there's something more automated, it may be best to include only large tickets (major version bumps) so that we are less likely to end up with stale data.
